packages/orchestration/src/types.d.ts

```diff
@@ -0,0 +1,7 @@
+import type { RouterProtocol } from '@agoric/network/src/router';
+
+export type ConnectionId = `connection-${number}`;
+
+export type AttenuatedNetwork = {
+  bind: RouterProtocol['bind'];
+};
```

Comment:
"Attenuated" sounds like it's meant to be a subset.
```suggestion
export type AttenuatedNetwork = Pick<RouterProtocol, 'bind'>;
```